hack/build/run-functional-tests.sh

```diff
@@ -72,6 +72,5 @@ if [ $retry_counter -eq $MAX_CDI_WAIT_RETRY ]; then
     exit 1
 fi
 
-test_command="${TESTS_OUT_DIR}/tests.test -test.timeout 180m ${test_args}"
-echo "${test_command}"
+test_command="${TESTS_OUT_DIR}/tests.test -test.timeout 360m ${test_args}"
```

Comment:
why 360m?

Reply:
With some failures 180 was not long enough any more. so I increased it